src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -68,14 +68,14 @@ public BoundLambda(SyntaxNode syntax, BoundBlock body, ImmutableArray<Diagnostic
         {
             if (inferReturnType)
             {
-                this._inferredReturnType = InferReturnType(
+                this._inferredReturnType = TypeSymbolWithAnnotations.Create(InferReturnType(
```

Comment:
>TypeSymbolWithAnnotations.Create [](start = 43, length = 32)

For some reason, every time I see `TypeSymbolWithAnnotations.Create` I wonder what it does.
Perhaps renaming the method to `CreateOblivious` would be clearer?

Reply:
Added a `// PROTOTYPE` comment to the method for now.

---
In reply to: [164233445](https://github.com/dotnet/roslyn/pull/23981#discussion_r164233445) [](ancestors = 164233445)